### ISSUE TITLE
The type or namespace name 'NullableAttribute' does not exist in the namespace 'System.Runtime.CompilerServices' 

### ISSUE THREAD

**Issue by yumikou** [postsharp/Metalama#27]
Hi! 
I am writing a wpf application and I want to insert a DependencyProperty field for class. 
Like this:
```
[DP(Name = "Custom", Type = typeof(string))]
public partial class TestClass
{
        public static readonly DependencyProperty CustomProperty = System.Windows.DependencyProperty.Register("Custom", typeof(System.String), typeof(TestClass));
}
```
This is my attribute:
```
public class DPAttribute : TypeAspect
{
        public string Name { get; set; }

        public Type Type { get; set; }

        public override void BuildAspect(IAspectBuilder<INamedType> builder)
        {
            builder.Advice.IntroduceField(
                builder.Target,
                nameof(DPAttribute.DependencyPropertyTemplate),
                buildField: f =>
                {
                    f.Name = $"{Name}Property";
                    f.Type = TypeFactory.GetType(typeof(System.Windows.DependencyProperty));
                    f.InitializerExpression = ExpressionFactory.Parse($"System.Windows.DependencyProperty.Register(\"{Name}\", typeof({Type}), typeof({builder.Target}))");
                }
                );
        }

        [Template]
        public static readonly dynamic? DependencyPropertyTemplate;
}
```

It work fine in the same project，but if I put DPAttribute in a Library，then build TestClass in the main project I get the following error:
```
Error	LAMA0611	Error CS0234 in code generated by aspect 'DP': The type or namespace name 'NullableAttributeAttribute' does not exist in the namespace 'System.Runtime.CompilerServices' (are you missing an assembly reference?)          TestClass.cs
Error	LAMA0611	Error CS0234 in code generated by aspect 'DP': The type or namespace name 'NullableAttribute' does not exist in the namespace 'System.Runtime.CompilerServices' (are you missing an assembly reference?)          TestClass.cs
```

I just started learning metalama, not sure if this is a bug or something wrong with my usage, 
If I change 'dynamic?' to 'dynamic' in Template of DPAttribute, it working well in TestClass of the main project,  should Field's Template use 'dynamic' instead of 'dynamic?' ?

the test project is here:
[IntroduceTest.WPF.zip](https://github.com/postsharp/Metalama/files/9192695/IntroduceTest.WPF.zip)

**Comment by gfraiteur** [comment]
Thank you for reporting this issue.
This is a bug of ours, there is nothing bad in your code.

**Comment by aweneer** [comment]
Hello @yumikou, we have recently fixed this bug in our newest released version of Metalama 0.5.32-preview.
